scalardb-analytics-postgresql-sample/README.md

```diff
@@ -147,20 +180,35 @@ Password for user postgres:
  c_comment    | text             |           |          |
 ```
 
-The column definitions are the same as the original table in the ScalarDB database. Type mapping between ScalarDB and PostgreSQL is explained in [Data type mapping between ScalarDB and the other databases](https://scalardb.scalar-labs.com/docs/latest/schema-loader/#data-type-mapping-between-scalardb-and-the-other-databases). Internally, this view is based on the foreign table explained above and interprets the transaction metadata to expose only the valid data with the Read Committed isolation level.
+The column definitions in this view are the same as the original table in the ScalarDB database. In ScalarDB, this view is based on the foreign table explained above and interprets the transaction metadata to expose only the valid data with the Read Committed isolation level.
```

Comment:
> In ScalarDB, this view is based on the foreign table explained above and interprets the transaction metadata to expose only the valid data with the Read Committed isolation level.

`In ScalarDB` is not correct, and I feel that this sentence is a bit difficult to understand. How about he following?

> This view is created based on the foreign table explained above to expose only the valid data with the Read Committed isolation level by interpreting the transaction metadata columns.

Reply:
Thank you! Sounds good to me. I've revised this as you suggested👍